sklearn/preprocessing/discretization.py

```diff
@@ -259,6 +299,15 @@ def inverse_transform(self, Xt):
             Data in the original feature space.
         """
         check_is_fitted(self, ["offset_", "bin_width_"])
+
+        # currently, preprocessing.OneHotEncoder
```

Comment:
Nit: The convention of this file would be to capitalize comments. ("currently" -> "Currently")